odpf/optimus/core/v1beta1/resource.proto

```diff
@@ -65,14 +66,16 @@ message DeployResourceSpecificationRequest {
 }
 
 message DeployResourceSpecificationResponse {
-  bool success = 1;
+  bool success = 1 [deprecated=true];
 
   // non ack responses are more of a progress/info response
   // and not success or failure statuses
-  bool ack = 2;
+  bool ack = 2 [deprecated=true];
 
-  string message = 3;
-  string resource_name = 4;
+  string message = 3 [deprecated=true];
+  string resource_name = 4 [deprecated=true];
+  Log log_status = 5;
+  Progress progress = 6;
```

Comment:
why is progress needed why can't it be in the status itself.

Reply:
`status log` has different purpose with `progress`. `status log` is used to send the log, while `progress` is used to send the progress count. We can't use `status log` in case where 1 operation send more than 1 logs